src/borg/archiver.py

```diff
@@ -4486,15 +4497,33 @@ def define_borg_mount(parser):
         change_algorithm_epilog = process_epilog("""
         Change the algorithm we use to encrypt and authenticate the borg key.
 
+        Important: In a `repokey` mode (e.g. repokey-blake2) all users share the same key.
+        In this mode upgrading to `argon2` will make it imbossible to access the repo for users that use an old version of borg.
+        We recommend upgrading to the latest stable version.
+
+        Important: In a `keyfile` mode (e.g. keyfile-blake2) each user has her own key (in ``~/.config/borg/keys``).
+        In this mode this command will only change the key used by the current user. Other users will still be able to access the repo.
```

Comment:
> "Other users will still be able to access the repo."

That sounds like they would not be able in the other modes, which can be true (if using an old borg), but is not necessarily true (if all use a new borg).

Considering that you have said that all users' keys need to be upgraded in the next sentence, it might be better to just leave away the quoted sentence.